### PR TITLE
meshnamed: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): *0.2.0*

### CODE REVIEW

File: pages/common/meshnamed.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# meshname
```
Comment (kbdharun) [inline pages/common/meshnamed.md:1]:
```suggestion
# meshnamed
```
Maybe, we could update the Heading to match with the filename and examples?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/meshnamed.md (hunk 1 above)
Comment (marek22k) [reply pages/common/meshnamed.md]: Thanks for the hint!

**Review by fazlearefin — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]
Thanks!

**Pull request merged**